win32ss/user/ntuser/ime.c

```diff
@@ -2461,12 +2475,14 @@ IntNotifyImeShowStatus(_In_ PWND pImeWnd)
         if (pWnd)
         {
             bSendNotify = TRUE;
+
+            ProbeForWrite(pimeui, sizeof(*pimeui), 1);
```

Comment:
This could be moved up together with the other ProbeForRead.
See also https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/wdm/nf-wdm-probeforwrite

Reply:
Joined.